_locales/it/messages.json

```diff
@@ -0,0 +1,300 @@
+	},
+
+	"extensionDescription": {
+		"message": "Apri un link nella barra laterale.",
+		"description": "Description of the extension."
+	},
+
+	"extensionName": {
+		"message": "Apri nella barra laterale",
+		"description": "Name of the extension."
+	},
+
+	"host": {
+		"message": "accedere ai dati dei siti web"
+	},
+
+	"iconImageAlt": {
+		"message": "L'icona di questa estensione.",
+		"description": "The alternate description of the icon of this extention."
+	},
+
+	"indexHTMLCaution": {
+		"message": "Nota: selezionando un url con protocollo https, sarà privilegiata l'opzione https anche se si sceglie http.",
+		"description": "Cautions of this extention."
+	},
+
+	"indexHTMLHeading1": {
+		"message": "1. Cliccare con il tasto destro del mouse su un link o la selezione di un url in formato testo e selezionare \"Apri nella barra laterale\" nel menu contestuale.",
+		"description": "The first heading of index.html."
+	},
+
+	"indexHTMLHeading2": {
+		"message": "2. Scegliere e cliccare sulla voce che si desidera aprire (usare https in caso di dubbi).",
+		"description": "The second heading of index.html."
+	},
+
+	"indexHTMLOthers": {
+		"message": "Ad esempio si possono aprire nella barra laterale un audio, un'immagine, un video, un segnalibro o una scheda.",
+		"description": "others of index.html"
+	},
+
+	"indexHTMLTitle": {
+		"message": "Guida",
+		"description": "The title of index.html."
+	},
+
+	"indexHTMLTutorialVideo": {
+		"message": "https://drive.google.com/file/d/14tn0lZpNIybOt94NmpdqJvWPlKlnLjCm/view",
+		"description": "The url of tutorial video."
+	},
+
+	"initialLocation": {
+		"message": "Pagina iniziale"
+	},
+
+	"initialLocationDescription": {
+		"message": "Premere 'Invio' per salvare. Se lasciato vuoto si resetta allo stato predefinito."
+	},
+
+	"openFromBookmark": {
+		"message": "Apri da un segnalibro"
+	},
+
+	"openFromLink": {
+		"message": "Apri da un link"
+	},
+
+	"openFromPage": {
+		"message": "Apri da una pagina"
+	},
+
+	"openFromSelection": {
+		"message": "Apri da una selezione"
+	},
+
+	"openFromSidebar": {
+		"message": "Apri dalla barra laterale "
+	},
+
+	"openFromTab": {
+		"message": "Apri da una scheda"
+	},
+
+	"openOptions": {
+		"message": "Apri le opzioni"
+	},
+
+	"openThisAudio": {
+		"message": "Apri questo audio"
+	},
+
+	"openThisBookmark": {
+		"message": "Apri questo segnalibro"
+	},
+
+	"openThisImage": {
+		"message": "Apri questa immagine"
+	},
+
+	"openThisLink": {
+		"message": "Apri questo link"
+	},
+
+	"openThisPage": {
+		"message": "Apri questa pagina"
+	},
+
+	"openThisSelection": {
+		"message": "Apri questa selezione"
+	},
+
+	"openThisSidebar": {
+		"message": "Apri questa barra laterale"
+	},
+
+	"openThisTab": {
+		"message": "Apri questa scheda"
+	},
+
+	"openThisVideo": {
+		"message": "Apri questo video"
+	},
+
+	"openTutorial": {
+		"message": "Apri la guida"
+	},
+
+	"optionsHTMLTitle": {
+		"message": "Optioni",
+		"description": "The title of options.html."
+	},
```

Comment:
Google Translation suggests "Opzioni" instead of "Optioni". It seems Opzioni is correct because you have translated option into opzioni in the line 121.